app/front/scripts/services/os-viewer/index.js

```diff
@@ -30,8 +30,11 @@ function getHierarchiesWithLimitedDimensionValues(hierarchies, minValueCount,
     .value();
 }
 
-function loadDataPackages(authToken) {
-  return dataPackageApi.getDataPackages(authToken);
+function loadDataPackages(authToken, packageId, userId) {
+  if (!packageId && !userId) {
+    new Promise.resolve([]);
```

Comment:
As `os-viewer-service` is a service that encapsulates high-level app logic - I decided to put here this condition: it can load only all users packages, or concrete package specified by ID; otherwise it should not load anything. The underlying API (`data-package-api` service) is more low-level and it should just load some datapackages. And then, controller will handle only general situation - if there are some packages - show them; otherwise - redirect to OS Explorer.